.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+	"bracketSpacing": false,
+	"printWidth": 80,
+	"singleQuote": true,
+	"useTabs": true
```

Comment:
@giotramu this must be removed (`useTabs` must be `false` and `false` is the default value)

Reply:
Mmh... sorry, I messed up with another open project.

Reply:
Done.